### PR TITLE
Mobile version

### PR DESCRIPTION
**Pull request by 29td**
Added some pages to html
made changes to style.css
installed linters

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,41 +1,191 @@
...
 <html lang="en">
 <head>
     <meta charset="UTF-8">
     <meta http-equiv="X-UA-Compatible" content="IE=edge">
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
-    <title>My Portfolio</title>
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com" crossorigin>
+    <link href="https://fonts.googleapis.com/css2?family=Inter&family=Roboto:wght@100&display=swap" rel="stylesheet">
     <link rel="stylesheet" href="/css/style.css">
```
Comment (Maclenn77) [inline index.html]: There is a problem when loading your CSS Stylesheet. I opened  the file in Firefox and Chrome and the stylesheet doesn't load. Consider the following change:

C
```suggestion
    <link rel="stylesheet" href="css/style.css">
```

**Review by Maclenn77 — CHANGES_REQUESTED** [review]
Hi @29td , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there! 

## Highlights

- You use correctly semantic tags. That means a lot! :muscle:

## Required Changes ♻️
_Check the comments under the review._

- [Mandatory] Constrain to Figma Template for the design . 

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:

- [OPTIONAL] Update Readme. Currently, your readme is a copy of Linters configuration instead of a description of the project. Consider changing to one that describes better the project. A professional readme ever stands out.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
It works if you set it to mobile instead of desktop version
Changes were made

File: index.html [hunk 2]
```diff
@@ -1,41 +1,191 @@
...
+    <section>
+        <div class="home-content">
+            <h1>Hey There. <br>I’m Nonhlanhla Mndebele </h1>
+            <h2>I’am a Software Developer</h2>
+            <p>I can help you build a product, feature or website look through some of my work and experience! If you
+                like what you see and have a project you need coded, don’t hesitate to contact me.</p>
+            <ul class="social">
+                <li class="github-a">
+                    <a href="#">
+                        <img src="/assets/icons/git.png" alt="img">
```
Comment (jaflih) [inline index.html]: All images are not loading, please correct it by adding a `.` before `/assets`

` 
img
`
Add '.'
` 
img
`

image

File: css/style.css [hunk 3]
```diff
@@ -1,96 +1,355 @@
...
+    flex-grow: 0;
+  }
+
+  .recent-works {
+    width: 100%;
+    margin: 0 24px;
+  }
+
+  .recent-works h1 {
+    font-family: 'Crete Round', sans-serif;
```
Comment (jaflih) [inline css/style.css]: Please use the good font to stick to Figma

image

File: css/style.css [hunk 4]
```diff
@@ -1,96 +1,355 @@
...
+    height: 48px;
+    align-items: center;
+    text-decoration: none;
+    color: white;
+    font-weight: 700;
+    font-size: 17px;
+    line-height: 24px;
+    font-family: 'Inter', sans-serif;
+    font-style: normal;
+    letter-spacing: 0.03em;
```
Comment (jaflih) [inline css/style.css]: Please add hover effect, as asked in the requirement

image

image

File: index.html [hunk 5]
```diff
@@ -1,41 +1,191 @@
...
+                    <img src="/assets/images/kite.png" alt="img">
+                    <h1>Language</h1>
+                </div>
+                <ul>
+                    <li>Javascript</li>
+                    <li>Ruby</li>
+                    <li>Html</li>
+                    <li>Css</li>
+                </ul>
+                <div class="title">
```
Comment (jaflih) [inline index.html:160]: Good job for using ul/li tag .
For the same reason, you should use it for Language/framework/skills

File: index.html [hunk 6]
```diff
@@ -1,41 +1,191 @@
...
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
+        </div>
+        <div class="about-wrapper">
+            <div class="about-me">
```
Comment (jaflih) [inline index.html:139]: Please update your design to stick to Figma
image

image

**Review by jaflih — CHANGES_REQUESTED** [review]
Hi @29td 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.




 ## Highlights

You use correctly semantic tags. That means a lot! 💪

 ## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
I've made changes

**Comment by 29td** [comment]
@Maclenn77
Changes were made below is my GitHub repo
https://github.com/29td/My-First-Portfolio/pull/2

File: css/style.css [hunk 7]
```diff
@@ -1,96 +1,357 @@
...
+  }
+
+  .work-item-alt {
+    width: 100%;
+    background-image: url('/assets/images/bg.png');
```
Comment (Div685) [inline css/style.css]: - [x] [Required] Please make sure all the images are rendered properly. This URL doesn't render Images. It's always a good practice to check your links and images before submission. 
You can use `background-image: url('../assets/images/bg.png');` to render image. 
image_Screenshot from 2022-05-29 01-28-15

**Review by Div685 — CHANGES_REQUESTED** [review]
Hi, **@29td**,  I'm Divyesh your reviewer for this PR :man_technologist: 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet. You are so Close to completing this milestone Don't give up :muscle: 


 ## Highlights
- :heavy_check_mark: No Linter Error
- :heavy_check_mark: Use of Semantic HTML

 ## Required Changes ♻️
**_Check the comments under the review._**
- [x] [Required] Please, make the necessary changes requested by the previous CR.
      - [ ] [Required] Please make sure all your images are rendering
      - [ ] [Required]  Please stick to the Figma Design as much as possible, especially with the Alignments, Padding, Margin, and Fonts. 

## Optional suggestions
- [x] [Optional] As requested by the previous CR, It's always a best practice to update your Resume as per your project description. It gives an overview of your project without even checking the code or live version.

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Code review by **Divyesh Patel**

_(Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.)_

------

>**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

>_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s).

File: index.html [hunk 8]
```diff
@@ -1,41 +1,187 @@
...
+                <h1>Profesional Art Printing Data</h1>
+                <p>A daily selection of privately personalized reads; no accounts or sign-ups required. has been the
+                    industry’s standard</p>
+                <ul>
+                    <li>html</li>
+                    <li>Bootstrap</li>
+                    <li>Ruby</li>
+                </ul>
+                <a href="#">See Project</a>
+            </div>
```
Comment (VICTOR-CHUKWU) [inline index.html]: - [x] Kindly add padding to your `work-item-alt` class so as to look exactly like the Figma design
   Your design :point_down: 
       
Screenshot from 2022-05-31 11-55-29
  Figma design :point_down: 
    
Screenshot from 2022-05-31 12-00-01
- [x] also align the `see project` tag to the center

File: index.html (hunk 6 above)
Comment (VICTOR-CHUKWU) [reply index.html:139]: - [x] Just as previous reviewer has said kindly Update your design as it has not yet been update

**Review by VICTOR-CHUKWU — CHANGES_REQUESTED** [review]
Hi @29td  👋  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

almost

## Highlights
- Good PR
- Good commit
- All linters passed


## Required Changes ♻️

_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
@VICTOR-CHUKWU I've made changes

File: index.html (hunk 8 above)
Comment (sja-thedude) [reply index.html]: - As suggested previously kindly update your design correctly as follows:
- [x] Make sure to remove the left white margin from the about section and languages section as shown in the video attached.
- [x] Kindly display your cards they are not visible.
- [x] Kindly follow the spacing in the languages section as per the Figma design.

Your design:

https://user-images.githubusercontent.com/70269507/171223553-30b76412-6bd3-4590-9afe-b7bc3a975b62.mov

Figma design:

https://user-images.githubusercontent.com/70269507/171223498-b4e5758b-1b9b-404f-883a-4abc74a8a181.mov

**Review by sja-thedude — CHANGES_REQUESTED** [review]
## Required Changes ♻️

Hi @29td, This is @sja-thedude your code reviewer.

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.

200w

 ## Highlights
✔️ Good job so far

 ## Required Changes ♻️
_Check Comments under the review_

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me [@sja-thedude] in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
Thank you for the feedback
I didn't realise it wasn't showing up on your side can you please help me
out because on my side everything seems okay even the pictures are visible
so I don't understand what could be the problem.

**Comment by sja-thedude** [comment]
> Thank you for the feedback I didn't realise it wasn't showing up on your side can you please help me out because on my side everything seems okay even the pictures are visible so I don't understand what could be the problem.

Hi @29td, Your images are not displayed because of CSS. Kindly add `../ `for all background images and your images will be displayed. 
Like for example: background-image: url('`../`assets/images/bg.png');
The part I highlighted have to be updated for all background images in CSS.

Also kindly update your readme with a professional readme template provided by microverse. 

## If you still need any help or have doubts just slack me on `@SJA`

**Comment by 29td** [comment]
Thank you so much for that
I've made changes hope my background picture is visible now
https://github.com/29td/My-First-Portfolio/pull/2

**Comment by 29td** [comment]
@*sja-thedude *

*Hey I've made changes but I haven't received any feedback till now*

**Comment by AshakaE** [comment]
Hi @29td , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by 29td** [comment]
***@***.****
Changes were made

**Comment by 29td** [comment]
@jaflih
I've made changes
https://github.com/29td/My-First-Portfolio/pull/2

**Pull request merged**